core/templates/dev/head/components/OutcomeDestinationEditorDirective.js

```diff
@@ -72,7 +72,8 @@ oppia.directive('outcomeDestinationEditor', [function() {
           // graph.
           var lastComputedArrangement = (
             StateGraphLayoutService.getLastComputedArrangement());
-          var allStateNames = Object.keys(explorationStatesService.getStates());
+          var allStateNames = Object.keys(
```

Comment:
Seems like we want this to be explorationStatesService.getStateNames() which in turn calls states.getStateNames().